cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForShuntCompensators.java

```diff
@@ -95,6 +100,21 @@ private void setRegulatingControl(ShuntCompensator shuntCompensator, RegulatingC
         }
         // Take default terminal if it has not been defined in CGMES files (it is never null)
         shuntCompensator.setRegulatingTerminal(parent.getRegulatingTerminal(shuntCompensator, rc.cgmesTerminal));
+
+        shuntCompensator.setProperty("RegulatingControl", rcId);
+    }
+
+    private void setAdditionalProperties(ShuntCompensator shuntCompensator, String rcId) {
+        if (rcId == null) {
+            return;
+        }
```

Comment:
Does it happen when this method is called?

Reply:
yes it could happen